FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Core.UnitTests/Jobs/FhirToDataLakeOrchestratorJobTests.cs

```diff
@@ -325,6 +336,8 @@ private static MockQueueClient GetQueueClient(FilterScope filterScope = FilterSc
                         SearchCount = new Dictionary<string, int> { { "Patient", 1 } },
                         ProcessedCount = new Dictionary<string, int> { { "Patient", 1 } },
                         SkippedCount = new Dictionary<string, int> { { "Patient", 0 } },
+                        ProcessedCountInTotal = 1,
+                        ProcessedDataSizeInTotal = 1000,
```

Comment:
Can we add some tests for really big size like 1T, 10T, 100T?

Reply:
Changed it to 1T.